native/tx.go

```diff
@@ -0,0 +1,49 @@
+package native
+
+import (
+	"context"
+	"crypto/ed25519"
+
+	"github.com/block-vision/sui-go-sdk/models"
+	"github.com/block-vision/sui-go-sdk/sui"
+	tmtypes "github.com/cometbft/cometbft/types"
+)
+
+func callMoveFunction(
+	ctx context.Context,
+	c *sui.Client,
+	lcpackage, module, function, gasbudget,
+	signerAddress, gasAddr string,
+	lb *tmtypes.LightBlock,
+) (models.TxnMetaData, error) {
+	return c.MoveCall(ctx, models.MoveCallRequest{
+		Signer:          signerAddress,
+		PackageObjectId: lcpackage,
+		Module:          module,
+		Function:        function,
+		TypeArguments:   []interface{}{},
+		Arguments: []interface{}{
+			lb,
+		},
+		Gas:       gasAddr,
+		GasBudget: gasbudget,
+	})
+}
+
+func executeTransaction(
```

Comment:
Create one function and run `executeTransaction` and `callMoveFunction` inside this function. This much be PeraClient methods.

Reply:
We have a function call `lcUpdateCall` inside `IkaClient`